### ISSUE TITLE
GLOSARIO

### ISSUE THREAD

**Issue by joaquinelio** [javascript-tutorial/es.javascript.info#225]
Great team presents:
--------------------
```
// haganlo lindo
// inserten alfabéticamente
// formato , no respeta tab
```

### arguments   argumentos, arguments  
OJO puede referirse a la forma específica de obtener los argumentos de una función, 
un array de argumentos. 
ej: mostrar(...arguments)   va on los 3 puntos

### gotcha
Cazabobos, trampa  (Got you!)

### key
key, clave.  Depende del contexto.
Es una clase *especial* de propiedad con la que se accede a un valor.
                
### shallow         
superficial, pero si se refiere a:

### shallow-clone (y deep-clone)   
clon-superficial  
prefiero el inglés con guion incluido, shallow-clone y deep-clone son dos formas de copiar un objeto.
PERO no es una definición JS, asi que hagan lo que se les cante.

### wrapper        
 wrapper, envoltura, contenedor. ¿Según contexto? 
Es un objeto que envuelve a otro para agegarle funcionalidad o alterar su comportamiento.
Si se traduce se dificulta conjugar uno u otro y hay que alternarlos.
Prefiero el inglés porque las bibliotecas de objetos estan llenos de wrappers,
                             
### zzz
zzz
Universal. Que me voy a dormir.

**Comment by ezzep66** [comment]
No me parece acertado la frase "Términos que no serán traducidos:". Me parece muy autoritaria.

Creo que cada traductor puede utilizar libremente la palabra **matriz, arreglo o vector**. Son términos que hacen referencia a lo mismo y se usan indistintamente. Si bien en la comunidad se utiliza comúnmente array, no debería haber una regla que no nos permita usar una palabra que CLARAMENTE es un sinónimo y no cambia el sentido de lo que queremos expresar.

En este caso existe el **objeto Array** que es un **objeto incorporado** al lenguaje. 
Esto **NO** se debe traducir. 

No es un capricho mio, lo he visto en distintos tutoriales.

[MDN Array](https://developer.mozilla.org/es/docs/Learn/JavaScript/First_steps/Arrays)

> Arreglos o Matrices

[tutorial](https://uniwebsidad.com/libros/javascript/capitulo-3/tipos-de-variables)

> 3.2.3. Arrays
En ocasiones, a los arrays se les llama vectores, matrices e incluso arreglos. 

No somos una comunidad autoritaria, somos una comunidad open source, todos colaboran, todos opinan, todos intercambian ideas. 


Otra cosa: **"Gotcha"** no es un error tipográfico. Por lo que yo interpreto, gotcha es una "trampa" del lenguaje, un bug. Me parece que la mejor traducción que encontré fue "trampa", ya que hace referencia que un resultado inesperado y sorpresivo.

**Comment by joaquinelio** [comment]
No maintainers, pueden hacer sugerencias acá como comentarios.

Si empieza a crecer desgloso glosario =)
separo glosario técnico de las sugerencias de palabras comunes.

**Comment by EzequielCaste** [comment]
No queda claro la sección Links. Mi sugerencia:

### Links internos:  

`[regular expression](info:regular-expressions)` >> se podrá traducir sólo la parte entre corchetes `[ ]`, el contenido entre paréntesis no modificar.

`<info:string>` >> No se traduce

`[#anchor]`  >> No se traduce

### Links externos:

Si el link es a Wikipedia o MDN y existe una versión en Español de calidad decente y aceptable, se podría cambiar el link a la versión ES.

**Comment by joaquinelio** [comment]
Sí, terrible... fue otro bosquejo agendado para depurar "más adelante"

Lo gracioso... es que lo quisse esscribir para sugerir que no perdieran tiempo en traducir mdn...
Si ven links a spec o mdn, ignoren y igan adelante.

**Comment by EzequielCaste** [comment]
que te parece si incluimos mi sugerencia?

**Comment by maksumi** [comment]
@EzequielCaste yo estoy de acuerdo pero sugiero "el contenido entre paréntesis no se debe modificar".

**Comment by joaquinelio** [comment]
si ok, esta bueno, ponelo 
es que me acorde que no es glosario, tenemos la guia de traduccion que ya  algo de eso comentaba y no supe si seguir aca y deje solo lo de mdn... 

que esté en los dos lados no hace mal tampoco,

**Comment by joaquinelio** [comment]
ALEX????

ey, te estaba por mandar un wasáp!!
todo bien?

**Comment by joaquinelio** [comment]
helper functions? 
en algun lado lo vi como "funciones de soporte",
son funciones que ayudan a otras funciones asi que "de ayuda" tampoco estaria mal
agregarelo quien hubierelo manejado en comunidad  plis

**Comment by joaquinelio** [comment]
**BUILT-IN**   rediscutirla ? 
"incorporada" 
es lo que usamos hasta ahora, seria "correcto" aunque en mi opinion muy facil de malinterpretar, las bibliotecas de 3ros tambien incorporan funciones
"integrada" 
me gusta más, no se por qué, me da idea de que ya esta ahi, no incorporado posteriormente en una biblioteca extra... "built-in" ="integrada" me parece muy bien
"nativa" me parecia perfecta pero no lo habiamos ussado antes ni la discutimos, podria haber desacuerdos,

**Comment by joaquinelio** [comment]
deje **lookahead  /behind /around** en ingles, 
* la version español es incomodamente larga
* porque el nombre no importa, mayormente se usa y es mas claro el simbolo
* dif regiones leen como fea una u otra traduccion y no hubo acuerdo
* MDN lo tradujo de manera inconsistente y con un problema gramtical
